src/components/index/page-sections/ServiceSection.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import IndexGradientBorderButtonBasic from '../../buttons/gradient-border-buttons/IndexGradientBorderButtonBasic';
+import ServiceSectionCard from './ServiceSectionCard';
+
+const ServiceSection = () => {
+  return (
+    // Section Container
+    <div className="container mx-auto mb-40">
+      <h1 className="flex flex-row w-full justify-center md:justify-start text-4xl md:text-5xl mb-12">
+        <FormattedMessage id="index.services.title" />
+      </h1>
+      {/* 3 COL GRID @SMALL SCREEN 1 COL */}
+      <div className="grid grid-cols-1 md:grid-cols-3 gap-10">
+        {/* 1. Service Content Column */}
```

Comment:
glaube dass du wirklich nicht so deep kommentieren musst. Der code ist ja auch sichtbar für andere, die den code lesen wollen (html oder repo)

Reply:
Können wir hier nochmal darüber sprechen welche Funktion gatsby in dem Kontext genau hat und was letztlich ausgeliefert wird und was nicht? :)

Reply:
mh würd mich wundern wenns raus gekickt wird. - bin ich mir aber auch gerade nicht sicher.